addons/website_slides/static/src/js/slides_course_tag_add.js

```diff
@@ -357,11 +19,14 @@ publicWidget.registry.websiteSlidesTag = publicWidget.Widget.extend({
      */
     _onAddTagClick: function (ev) {
         ev.preventDefault();
-        this._openDialog($(ev.currentTarget));
+        const props = {
```

Comment:
I'm still not convinced by this very short-lived variable. If you really like it though, we should at least give it a more explicit name like `addTagDialogProps` imho.

Reply:
ok, I replace props by an anoymous object.